src/backend/TrafficCourts/Staff.Service/Mappers/Mapper.cs

```diff
@@ -18,37 +19,11 @@ public static DisputeApproved ToDisputeApproved(Dispute dispute)
         target.IssuingOrganization = dispute.ViolationTicket.DetachmentLocation;
         target.IssuingLocation = dispute.ViolationTicket.CourtLocation;
         target.DriversLicence = dispute.DriversLicenceNumber;
-        List <Messaging.MessageContracts.ViolationTicketCount> violationTicketCounts = new();
-        foreach (var violationTicketCount in dispute.ViolationTicket.ViolationTicketCounts)
-        {
-            Messaging.MessageContracts.ViolationTicketCount ticketCount = new()
-            {
-                Count = violationTicketCount.CountNo,
-                Subparagraph = violationTicketCount.Subparagraph,
-                Section = violationTicketCount.Section,
-                Subsection = violationTicketCount.Subsection,
-                Paragraph = violationTicketCount.Paragraph,
-                Act = violationTicketCount.ActOrRegulationNameCode,
-                Amount = violationTicketCount.TicketedAmount
-            };
-
-            violationTicketCounts.Add(ticketCount);
-        }
 
-        List<Messaging.MessageContracts.DisputedCount> disputeCounts = new();
-        foreach (var dc in dispute.DisputeCounts)
-        {
-            Messaging.MessageContracts.DisputedCount disputeCount = new()
-            {
-                Count = dc.CountNo,
-                DisputeType = nameof(dc.PleaCode)

```

Comment:
This was the main broken stuff. The DisputeType would have always come through as "PleaCode"